### DISCUSSION TITLE
React Aria Popover with Framer Motion

### DISCUSSION THREAD

**Discussion by endreujhelyi** [adobe/react-spectrum#7170] [Q&A] [answered]
When using `Menu` and `Popover` with Framer Motion, it doesn't trigger the exit animation, or any animation at all. It seems to get stuck on the first frame.

I tried following the Modal example described here without any luck: https://react-spectrum.adobe.com/react-aria/styling.html#framer-motion.

Here’s a sandbox for reference: https://codesandbox.io/p/sandbox/lkqfq2.

Any ideas on how to make it work with Framer Motion?

**Comment by snowystinger** [comment, marked as answer]
Hey, sorry for the delay, I'm not seeing any of the styles correctly in the sandbox. I also assume you meant to use `Button`, not `button` in it for the trigger?

All of that aside, have you tried this with the `Modal` and `ModalOverlay`'s directly as shown in the example you linked? That may give you more of the control necessary to do this.

Here's a working example https://react-spectrum.adobe.com/react-aria/examples/framer-modal-sheet.html
